bin/php-semver-checker-git

```diff
@@ -1,28 +1,18 @@
 #!/usr/bin/env php
 <?php
 
-$autoload_paths = array(
-	__DIR__.'/../vendor/autoload.php',
-	__DIR__.'/../../../autoload.php',
-);
-
-$found = false;
-foreach ($autoload_paths as $path) {
+foreach (array(
+             __DIR__.'/../vendor/autoload.php',
+             __DIR__.'/../../../autoload.php',
+         ) as $path) {
 	if (file_exists($path)) {
-		require $path;
-		$found = true;
-		break;
+		require($path);
```

Comment:
This is a bit unrelated to this PR.

Reply:
agreed, not quite sure why I committed this to this branch. Should I remove it?

Reply:
Yes. Please keep the PR contained :wink:.